File: sdk/core/azure-core/src/main/java/com/azure/android/core/util/paging/PageLoadState.java
package com.azure.android.core.util.paging;

/**
 * The page load state.
 */
public enum PageLoadState {
    /**
     * Page loading is in progress.
     */
    LOADING,
    /**
     * Page loading is completed.
     */
    LOADED,
    /**
     * Page loading is failed.
     */
    FAILED,
}

File: samples/sample-app-storage-kotlin/src/main/java/com/azure/android/storage/sample/kotlin/core/util/paging/PageLoadState.kt
package com.azure.android.storage.sample.kotlin.core.util.paging

/**
 * The page load state.
 */
enum class PageLoadState {
    /**
     * Page loading is in progress.
     */
    LOADING,

    /**
     * Page loading is completed.
     */
    LOADED,

    /**
     * Page loading is failed.
     */
    FAILED
}
